Makefile

```diff
@@ -39,3 +39,10 @@ configure:
 .PHONY: build doc test all install uninstall reinstall clean distclean configure
 
 # OASIS_STOP
+
+.PHONY: js-install js-uninstall
+js-install:
+	install -m 0644 js/cstruct.js $(shell ocamlfind query cstruct)
```

Comment:
could this shell invocation be stuck into a Make variable instead so it can be manually overridden?  I do worry about cross compilation these days.